vendor/glpk/glpenv03.c

```diff
@@ -45,9 +45,9 @@
 
 void glp_printf(const char *fmt, ...)
 {     va_list arg;
-      /* va_start(arg, fmt); */
-      /* xvprintf(fmt, arg); */
-      /* va_end(arg); */
+      va_start(arg, fmt);
+      xvprintf(fmt, arg);
```

Comment:
Would the R interface (or, well, more like the CRAN validation of the R interface) choke on the presence of a printf-like function?